### PR TITLE
Added migration file with data structure planning

### PR DESCRIPTION
**Pull request by vinnihoke**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts
- [ ] I have checked that no one else is on staging and deployed this feature branch

### CODE REVIEW

**Comment by vinnihoke** [comment]
This is getting a lot of code climate errors. Any advice so I can clean this up?

File: data/migrations/20200218211829_home-run-data.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+exports.up = function (knex) {
+	return knex.schema
+		.createTable('members', col => {
+			col.increments()
+			col.string('username', 40).notNullable()
+			col.string('email', 128).notNullable()
+			col.string('password').notNullable()
+			col.string('token')
+			col.string('current_household')
```
Comment (MicahJank) [inline data/migrations/20200218211829_home-run-data.js]: Isnt current household suppose to be a foreign key pointing to the household table?

Comment (zbtaylor) [reply data/migrations/20200218211829_home-run-data.js]: I believe you're correct. @vinnihoke

File: data/dbConfig.js [hunk 2]
```diff
@@ -1,6 +1,151 @@
 const knex = require("knex");
 const environment = process.env.NODE_ENV || "development";
 
-const configOptions = require("../knexfile")[environment];
+const configOptions = require("../knexfile").development;
```
Comment (zbtaylor) [inline data/dbConfig.js]: I think this needs to be changed back to `require("../knexfile")[environment]`

Comment (zbtaylor) [inline data/dbConfig.js]: I think this needs to be changed back to `require("../knexfile")[environment]`

**Pull request merged**